core/templates/dev/head/library/library.html

```diff
@@ -44,7 +44,15 @@
         </div>
 
         <div class="oppia-exp-summary-tiles-container" ng-if="!inSearchMode">
-          <img src="/images/library/banner.svg" align="center" width="100%">
+          <script type="text/javascript">
+            window.onload = chooseCloud;
+            var newCloud = new Array("/images/library/banner1.svg","/images/library/banner2.svg","/images/library/banner3.svg","/images/library/banner4.svg");
+            function chooseCloud(){
+            cloudNum = Math.floor((Math.random() * newCloud.length));
+            document.getElementById("iconCloud").src = newCloud[cloudNum]
+            }
+          </script>
+          <img id="iconCloud" align="center" width="100%">
```

Comment:
It would be better to do this using the existing AngularJS framework. Here's how:

In the HTML file, replace the above line with:

```
    <img ng-src="/images/library/<[bannerImageFilename]>" align="center" width="100%">
```

and remove all the JavaScript. Note that the ng-src attribute refers to a `<[bannerImageFilename]>` -- I'll explain this below.

Then, in the Library.js file, just below where `$rootScope.loadingMessage` is set, add a line like this:

```
    $scope.bannerImageFilename = 'banner1.svg';
```

and refresh the page. What happens when the page reloads is: $scope.bannerImageFilename is set to the string 'banner1.svg', and then the ng-src tag will take this variable and interpolate it into the string it has, in order to produce '/images/library/banner1.svg' as the src for the image URL. (If you can't get this to work, please let me know, and I'd be happy to help!)

Once you've got this working, see if you can figure out a way to set $scope.bannerImageFilename to either banner1.svg, banner2.svg, banner3.svg or banner4.svg at random. As a hint, the simplest way is probably to store all four filenames in an Array, like this:

```
    var possibleBannerImageFilenames = [
      'banner1.svg', 'banner2.svg', 'banner3.svg', 'banner4.svg'];
```

and set $scope.bannerImageFilename to equal a random selection from this array.

In passing, btw, note that the array is declared using "var" instead of "$scope", so that it's not exposed to the HTML template. We do this in order to keep things tidy and encapsulated, since there's no need to expose functionality where it isn't needed. Only $scope variables can be interpolated in the HTML using the `<[...]>` syntax.